go-controller/pkg/ovn/master.go

```diff
@@ -928,12 +928,19 @@ func (oc *DefaultNetworkController) deleteOVNNodeEvent(node *kapi.Node) error {
 		oc.syncZoneICFailed.Delete(node.Name)
 	}
 
+	// Remove management port IP and node's gateway-router-lrp-ifaddr
+	// from address_set specific to HostNetworkNamespace
+	if err := oc.delIPFromHostNetworkNamespaceAddrsSet(node); err != nil {
+		return err
```

Comment:
here and everywhere please put a meaningful error message that will be easy to debug in the future:
`return fmt.Errorf("failed to delete nodeIP %s from the address-set %s: err %v")`
easier to debug user issues in the future

Reply:
Noted, Thanks!